### DISCUSSION TITLE
Unable to connect Antmedia server to Redis database

### DISCUSSION THREAD

**Discussion by Samriddha01** [ant-media/Ant-Media-Server#5340] [Q&A]
In the Ant media server source code, from where we can do the connection to the Redis database? I just want to configure the Redis and send the data to the Redis database? How can i do the same in the Ant media Source code?

**Comment by lastpeony** [comment]
Check ConsoleDataStoreFactory.java
https://github.com/ant-media/Ant-Media-Server/blob/7fc0ec69c74ae4b603fcf902f99bf7259b6919b4/src/main/java/io/antmedia/console/datastore/ConsoleDataStoreFactory.java
As you see AMS reads database type from settings and initializes database according to that.

Also check RedisStore.java for connection details. You can edit this class according to your needs.
https://github.com/ant-media/Ant-Media-Server/blob/7fc0ec69c74ae4b603fcf902f99bf7259b6919b4/src/main/java/io/antmedia/datastore/db/RedisStore.java

**Reply by mekya** [reply]
Hi @Samriddha01,

There was a blog post about how to use redis and other databases
https://antmedia.io/databases-supported-by-ant-media-server/

If you want to change the source code and build the things yes the answer of @lastpeony helps you.

Please let us know if we can help you further.
